controllers/ovncontroller_controller.go

```diff
@@ -547,6 +558,8 @@ func (r *OVNControllerReconciler) generateServiceConfigMaps(
 	templateParameters := make(map[string]interface{})
 	if instance.Spec.NetworkAttachment != "" {
 		templateParameters["OvnEncapNIC"] = nad.GetNetworkIFName(instance.Spec.NetworkAttachment)
+	} else if contains(instance.Spec.NetworkAttachments, "tenant") {
```

Comment:
please put the hardcoded name into a const somewhere in the package, with an explanation what it is (maybe the name can be self-explanatory, e.g. `TenantNetworkAttachmentName` or `TunnelNetworkAttachmentName`)?

Reply:
ack done